tox.ini

```diff
@@ -1,15 +1,12 @@
 [tox]
-envlist = py27, py33, py34, py35, py36, pypy, lint
```

Comment:
Why did you remove PyPy from testing?

Reply:
I have re-added pypy, and also added pypy3 testing.